scripts/mailserver-launch-starter/mailserver-launch-starter.bash

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+echo "___________________________________"
+echo "*    MAIL SERVER SETUP SCRIPT     *"
+echo "___________________________________"
+
+# Load application.properties
+application_properties_dir="../../src/main/resources/application.properties"
+if [ -f "$application_properties_dir" ]; then
+    CONTAINER_NAME=$(grep "DOCKER.CONTAINER.NAME.MAILDEV" $application_properties_dir | cut -d'=' -f2)
+    USERNAME=$(grep "spring.mail.username"  $application_properties_dir  | cut -d'=' -f2)
+    PASSWORD=$(grep "spring.mail.password"  $application_properties_dir  | cut -d'=' -f2)
+    MAILDEV_OUTGOING_PORT=$(grep "MAILDEV_OUTGOING_PORT" $application_properties_dir  | cut -d'=' -f2)
+    MAILDEV_WEB_PORT=$(grep "MAILDEV_WEB_PORT" $application_properties_dir  | cut -d'=' -f2)
+
+
+    echo "SUCCESS: variables from application.properties are loaded."
+else
+    echo "Error: application.properties file not found."
+    exit 1
+fi
+
+# Checking if docker daemon is running
+if ! docker info > /dev/null 2>&1; then
+    echo "Error: Failed to connect to Docker daemon. Please check if Docker daemon is running and accessible."
+    echo "To solve this problem, try using the command depending on your system:"
+    echo "    -> sudo systemctl start docker"
+    echo "    -> sudo service docker start"
+    echo "and start script again."
+    exit 1
+fi
+
+# Check if the container already exists, if so, delete it
+if docker ps -a --format "{{.Names}}" | grep -q "^$CONTAINER_NAME$"; then
+    echo "Deleting old container: $CONTAINER_NAME"
```

Comment:
```suggestion
    echo "Deleting container: $CONTAINER_NAME"
```

Reply:
I can replace the word "old" with "previous" but I think it should be one of these two to know that it is about a previously created container not necessarily by this script